### ISSUE TITLE
incorrect relationships btw 2 complexes and GO:0000790 nuclear chromatin

### ISSUE THREAD

**Issue by bmeldal** [geneontology/go-ontology#13900]
GO:0070209 | ASTRA complex and
GO:0070211 | Snt2C complex
have is_a relationships to GO:0000790 nuclear chromatin.

That is clearly incorrect. They are chromatin remodeling complexes and should have part_of relationships to chromatin. 

Correct me if I'm wrong but I'm working on the SWI/SNF (BAF) family of chromatin remodeling complexes now and trying to assess their GO relationships (another ticket in progress!).

**Comment by krchristie** [comment]
Whether these complexes are 'part of' chromatin or not seems to depend on how chromatin is defined, whether loosely or tightly and we've debated that before at a GOC meeting. However, it's clearly wrong to say that these complexes are types of chromatin!

So, I'm fine with changing the relationships from 'is_a' to 'part of'.

I also happened to notice that the 'HDA1 complex' also had 'nuclear chromatin as an 'is_a' parent, so I'm changing that one as well.
